app/views/samples/index.haml

```diff
@@ -20,20 +20,23 @@
                 .notification-text= "QC Samples can't be transferred"
 
           - t.columns do
-            %col{:width => "5%"}
-            %col{:width => "20%"}
-            %col{:width => "20%"}
-            %col{:width => "20%"}
-            %col{:width => "20%"}
-            %col{:width => "20%"}
+            %col{:width => "4%"}
+            %col{:width => "12%"}
+            %col{:width => "18%"}
+            %col{:width => "12%"}
+            %col{:width => "26%"}
+            %col{:width => "13%"}
+            %col{:width => "15%"}
           - t.thead do
             %tr
               %th
               %th Sample Id
-              %th Isolate
+              %th sample_isolate_name
               %th Inactivation
               %th Specimen Role
               %th Production Date
+              %th= link_to "Modified", ( params["sort"] ? request.query_parameters.except(:sort) : request.query_parameters.merge({:sort => 'updated_at'}) ), :class => "#{params["sort"] ? 'ordered' : ''}"
```

Comment:
There is a little too much happening on a single line, and we can check the sort parameter's value to be future proof.

```suggestion
              - if params[:sort] == "updated_at"
                %th= link_to "Modified", request.query_parameters.except(:sort), class: "ordered"
              - else
                %th= link_to "Modified", request.query_parameters.merge(sort: "updated_at")
```